src/jobs/qa-automation-suite.yaml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  test_path:
+    description: "The path to the test suite to run"
+    type: string
+  project_name:
+    description: "The name of the project in report portal"
+    type: string
+  launch_name:
```

Comment:
is this a parameter for report portal? in case I suggest to call it `report_portal_name` or something alike

Reply:
I would rather keep the name and improve the description.
By using this orb you know that you are reporting to report portal, it seems redundant to "report_portal" to the name of the variables

Reply:
here some parameters are referring to the `qa-automation-test` suite itself and other to the RP integration.
At best of my knowledge you can disable the integration with `report_portal` (@jonathancarvalheiro  can you confirm?)
so
> By using this orb you know that you are reporting to report portal
is not correct

Reply:
It's not configurable, the only way it's not to set the RP parameters. I'll check if the docker image works without setting the RP parameters.

Reply:
So I checked, and it's mandatory to have the RP parameters defined.

Reply:
I purpose to leave like it is right now, and when I put this as a configuration, I do a PR in here as well.